### PR TITLE
Update Dockerfile to build dev on arm64

### PR DESCRIPTION
**Pull request by elgreg**
A few libraries (looks like node canvas and rocksdb) do not have arm64 binaries available. npm/yarn will fallback to try to build them from source, but it cannot without build-essential (make) and a group of other packages for node-canvas.

Automattic has documented Ubuntu dependencies in the [compiling section of their readme](https://github.com/Automattic/node-canvas#compiling)

This also moves the apt calls to be before yarn.

Steps to test:

1. retag your old dev, so that you still have it in case things go poorly `docker tag dev crosshare-dev:old` or some such.
2. docker-compose up dev

This build took a _very_ long time for me (10 minutes), so don't get discouraged if it is taking forever.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,9 +2,9 @@ FROM mcr.microsoft.com/playwright:v1.32.2-jammy as dev
 RUN mkdir /src
 WORKDIR /src
 COPY app/package.json app/yarn.lock ./
+RUN apt-get update && apt-get install -y --no-install-recommends openjdk-11-jre-headless curl build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```
Comment (mdirolf) [inline Dockerfile:5]: We could probably ditch libgif and libjpeg - what we use canvas for is rendering the puzzle preview images that populate the og:image metatag and we only use svg/png for that. But I doubt that will have any significant savings in image build time or anything so probably fine to keep.

**Comment by mdirolf** [comment]
This looks good. I'm just going to make sure it still builds fine for me and then will merge.

**Comment by elgreg** [comment]
Cool. Happy to remove those and make sure still builds if you'd like, but all the build time for me was in yarn anyway (almost 15 minutes!) I think b/c of the arm binary compilation steps.

**Comment by elgreg** [comment]
Also resolved the merge conflict from my other i18n pr.

**Comment by mdirolf** [comment]
> Cool. Happy to remove those and make sure still builds if you'd like, but all the build time for me was in yarn anyway (almost 15 minutes!) I think b/c of the arm binary compilation steps.

Yeah the dependency installation takes a long time for me too - I've just been operating under the assumption that that was some unavoidable consequence of working in the JS ecosystem 😂. I do try to avoid adding deps unless totally necessary but they still seem to balloon.

**Pull request merged**